conftest.py

```diff
@@ -1,29 +1,36 @@
+"""PyTest Fixtures."""
 import os
+from test import RunFromText
```

Comment:
This import is invalid in this place. It's not coming from stdlib.

Reply:
Hmm..., that is tricky because conftest.py is not really a python module so nobody is doing any from ansiblelint import conftest. These conftest files are special files used by pytest, which looks for all of them upwards. But anything defined in conftest files outside your paterns, will not be available to your test. This is why we need it at the root, to make them available for both the library and the tests, which are two different branches.

Probably we can get around by creating a symlink. Would that please you?

Reply:
No, this should be added to isort as local/first-party import and that will work just fine.